### DISCUSSION TITLE
Switch to the communitydriven ciscoisesdk?

### DISCUSSION THREAD

**Discussion by falkowich** [falkowich/ise#231] [General]
Hi,
I feel that my time with this library are really limited.
So I wonder it it's time to change to https://github.com/CiscoISE/ciscoisesdk ?

I found this the other day, and I think that is solve most of my itches.
How do you see this move?

If anyone want to take over this just say the word.
Otherwise I think that I will archive this repo, if no one have any opinion about this :)

--
Kind Regards Falk

**Comment by JonasKs** [comment]
I think this makes sense.😊 Thanks for maintaining this over the years.

**Comment by falkowich** [comment]
I'm just going to release 0.3 for the ISE 2.7 ppl.
From what I can see when messing around with ciscoisesdk it doesn't support the old 2.7 train.

So I think I'll release 0.3 and hang around until 2.7 is end of support.

https://www.cisco.com/c/en/us/products/collateral/security/identity-services-engine/bulletin-c25-2943876.html
